src/types.ts

```diff
@@ -22,20 +20,20 @@ export interface DateMap {
   [date: string]: State[];
 }
 
-export type Locale = string;
+export type CalendarLocale = 'es' | 'en';
```

Comment:
Couldn't you leave it as `string` and then `require('date-fns/locale/' + locale)`?